gulpfile.js/tasks/init.js

```diff
@@ -1,11 +1,11 @@
-var gulp = require('gulp')
-var log = require('fancy-log')
-var colors = require('ansi-colors')
+var gulp        = require('gulp')
+var log         = require('fancy-log')
+var colors      = require('ansi-colors')
 var projectPath = require('../lib/projectPath')
-var merge = require('merge-stream')
+var merge       = require('merge-stream')
```

Comment:
I'd argue against doing this sort of alignment. Adds unnecessary noise to PRs just for spacing changes.

Reply:
ah good to know. in the future i won't include these kinds of changes in PRs